src/adapter_covid19/README.md

```diff
@@ -23,22 +23,20 @@ python -m adapter_covid19.example
 
 Alternatively:
 ```python
-import itertools
-import numpy as np
-
-from adapter_covid19.datasources import Reader
-from adapter_covid19.corporate_bankruptcy import CorporateBankruptcyModel
-from adapter_covid19.economics import Economics
-from adapter_covid19.gdp import LinearGdpModel
-from adapter_covid19.personal_insolvency import PersonalBankruptcyModel
-from adapter_covid19.enums import Region, Sector, Age
-
-utilisations = {k: np.random.rand() for k in itertools.product(Region, Sector, Age)}
-econ = Economics(LinearGdpModel(), CorporateBankruptcyModel(), PersonalBankruptcyModel())
-econ.load(reader)
-for t in range(10):
-    econ.simulate(time=t, lockdown=False, utilisations=utilisations)
-# inspect econ.results
+from adapter_covid19.data_structures import Scenario, ModelParams
+from adapter_covid19.simulator import Simulator
+
+# Define data path
+data_path = f"./tests/adapter_covid19/data"
+
+# Initialize simulator
+simulator = Simulator(data_path)
+
+# Define your senarior
+scenario = Scenario()
+
+# Run simulation
+eco, states = simulator.simulate(scenario)
```

Comment:
Removed pickles and restructured

Reply:
@al626 Thanks - the line `from examples.example_run_spread_model_for_economics import run` in the readme now breaks because there is a utilities module being used by the imported module (it is not installed and prefer not installing). So I think the quickest way to fix this is to not use the utilities module (the functions therein are quite simple, you can just replicate their contents in `example_run_spread_model_for_economics`).

Reply:
@al626 Thanks - the line from examples.example_run_spread_model_for_economics import run in the readme now breaks because there is a utilities module being used by the imported module (it is not installed and prefer not installing). So I think the quickest way to fix this is to not use the utilities module (the functions therein are quite simple, you can just replicate their contents in example_run_spread_model_for_economics).

Reply:
Done